backend/Match/models.py

```diff
@@ -10,10 +10,9 @@ class Match(models.Model):
 	start_time = models.DateTimeField(null=True, blank=True)
 	end_time = models.DateTimeField(null=True, blank=True)
 	created_at = models.DateTimeField(auto_now_add=True)
-	updated_at = models.DateTimeField(auto_now=True)
+	update_at = models.DateTimeField(auto_now=True)
```

Comment:
Just for my own understanding: Why is created at in past tense and update in present tense?